### PR TITLE
double division?

### PR DESCRIPTION
**Pull request by TTKettler**

### CODE REVIEW

File: divide.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-def integer_division(dividend, divisor):
-    return dividend // divisor
+def double_integer_division(dividend, divisor):
```
Comment (robsidian) [inline divide.py:1]: Could you add some documentation here?

Comment (TTKettler) [reply divide.py:1]: yes i can!

**Comment by zrxdaly** [comment]
this looks good, i would love to see the end product

**Review by robsidian — APPROVED** [review]
all good

**Pull request merged**